src/sidebar/templates/top-bar.html

```diff
@@ -34,11 +34,11 @@
     <group-list class="group-list" auth="vm.auth"></group-list>
     <div class="top-bar__expander"></div>
     <a class="top-bar__apply-update-btn"
-       ng-if="vm.pendingUpdateCount > 0"
-       ng-click="vm.onApplyPendingUpdates()"
+       ng-if="vm.pendingUpdateCount() > 0"
+       ng-click="vm.applyPendingUpdates()"
```

Comment:
I noticed you changed the name here-just curious what is the naming convention for this kinda thing? Seems like sometimes we use `on` and sometimes we don't.

Reply:
This file has been removed now since the `<top-bar>` component was ported to Preact.

We generally use `on<Event>` naming for component inputs that are callbacks called in response to some abstract event (eg. "user clicked Sign Up"). In contrast functions / methods created within a component to perform a specific task and that are used as event handlers are generally named after what the function does.

This means that if you see something like `<button onClick={onSignUpClicked}>Sign up</button>` you can see at a glance that the event is propagated up to be handled by the parent, whereas if you see `<button onClick={showSignUpForm}>Sign up</button>` you know the event is locally handled (in this case to show a form).